mautrix_instagram/web/provisioning_api.py

```diff
@@ -782,6 +782,9 @@ async def post_fb_login_token(self, request: web.Request) -> web.Response:
             return self._2fa_required(user, "<facebook credentials>", state, e)
         except IGCheckpointError as e:
             return self._checkpoint_error(user, "<facebook credentials>", e, after="facebook auth")
+        except IGChallengeError as e:
+            self.log.debug("%s logged in as %s, but got a challenge", user.mxid, username)
```

Comment:
Username isn't known at this point

```suggestion
            self.log.debug("%s logged in with facebook credentials, but got a challenge", user.mxid)
```